src/components/Mandal/MandalPage.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import { useDispatch, useSelector } from "react-redux";
-import MainMandal from "./MainMandal";
+import MainMandal from "./view/MainMandal";
 import styled from "styled-components";
-import { changeToMainGoal, chageToFullView } from "../features/viewSlice";
-import { changeEditMode } from "../features/editSlice";
+import { changeToMainGoal, chageToFullView } from "../../features/viewSlice";
```

Comment:
```suggestion
import { changeToMainGoal, changeToFullView } from "../../features/viewSlice";
```

요거 슬라이스도 이름 수정 필요하겠습니다. 👀